commands-yml/commands/interactions/touch/tap.yml

```diff
@@ -26,12 +26,7 @@ example_usage:
       await action.perform();
   javascript_wdio:
     |
-      driver.touchPerform({
-        action: 'tap',
-        options: {
-          element: element
-        }
-      });
+      <element>.touchAction('tap');
```

Comment:
Could you remain driver.touchAction, too?
https://webdriver.io/docs/api/browser/touchAction.html

Reply:
Changed `<element>` to `driver`. Thanks.